katalog/thanos/thanos-components/thanos-store/thanos-storage-secret.yaml

```diff
@@ -0,0 +1,7 @@
+type: S3
+config:
+  bucket: "bucket-name"
+  endpoint: "s3.eu-west-5.amazonaws.com"
+  #encrypt_sse: true
+  access_key: "AKIAIOSFODNN7EXAMPLE"
+  secret_key: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
```

Comment:
should the user change these values? If that's the case instructions to do that should be added on the readme.

Reply:
is an example referenced by the README.md, have you a better idea on how handling this kind of things?

Reply:
Sorry, I missed somehow the reference in the README. Marking conversation as resolved.